### DISCUSSION TITLE
[Podman 5.0 Fedora 40 Beta] Container 2 Container without podman network cant connect over host.containers.internal:8080

### DISCUSSION THREAD

**Discussion by chrismaster** [containers/podman#22235] [Q&A]
On a fresh Fedora 40 Server Netinstall I try to:
Container A: podman run -p 8080:80 nginx
Container B;  podman run -it alpine sh
Container B: apk add curl
Container B: curl host.containers.internal:8080 or curl pubIP:8080 gives an error: curl: (7) Failed to connect to host.containers.internal port 8080 after 0 ms: Couldn't connect to server
I can curl from Container B to google and it works
I can curl from host to pubIP:8080 and it works
But I can't get a connection from one container to another.
Is there some magic selinux or firewall stuff I have to enable?
I don't want to put the containers into a pod or into the same podman network.
Thx
Chris

**Comment by chrismaster** [comment]
OK, maybe someone gets into same trouble.
edit
/usr/share/containers/containers.conf
and add eg
pasta_options = ["-a", "10.0.2.0", "-n", "24", "-g", "10.0.2.2", "--dns-forward", "10.0.2.3"]
see
https://blog.podman.io/2024/03/podman-5-0-breaking-changes-in-detail/

**Reply by meldafert** [reply]
Thanks, the blog post is really helpful! In my situation, using 
```
[network]
default_rootless_network_cmd = "slirp4netns"
```
also worked.
However, I think it'd be better to add this to `/etc/containers/containers.conf` - things in `/usr/share` are usually not supposed to be manually edited afaik (might be overwritten on the next update).
